### PR TITLE
docs: add user-focused content to readme

### PR DESCRIPTION
**Pull request by stdavis**
@steveoh Should this be a blog post? After I got going, I wondered if it should be in the website with a link to the GitHub repo similar to the API Client. I would prefer to keep it closer to the code in the readme, but I'm wondering if there's a reason why you did it that way with the API client.

### CODE REVIEW

**Review by steveoh — COMMENTED** [review]
I like it. We should do both. Not sure why we didn't give the read me some love on the api client. Is the issue that's causing the gcp monitor emails fixed?

**Comment by stdavis** [comment]
> Is the issue that's causing the gcp monitor emails fixed?

It looked like to me the monitor emails are from errors querying open sgid. They didn't look prevelent enough to me to prevent a public release. I was going to ask you what you thought about me adding some retries.

**Comment by stdavis** [comment]
This is ready for a look. I'll copy the majority of this to a blog post on gis.utah.gov once it's merged.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,91 @@
...
+
+![addresses](./screenshots/addresses.png)
+
+![cities](./screenshots/cities.png)
+
+![place names](./screenshots/place-names.png)
+
+![political districts](./screenshots/political-districts.png)
+
+Once Masquerade has been added as a locator to your project, you can also use it as an input address locator with the [Geocode Addresses geoprocessing tool](https://pro.arcgis.com/en/pro-app/latest/tool-reference/geocoding/geocode-addresses.htm) for batch geocoding of address data. This makes Masquerade a much easier alternative to the our old [Geocoding Toolbox](https://github.com/agrc/geocoding-toolbox).
```
Comment (steveoh) [inline README.md:65]:
```suggestion
Once Masquerade has been added as a locator to your project, you can also use it as an input address locator with the [Geocode Addresses geoprocessing tool](https://pro.arcgis.com/en/pro-app/latest/tool-reference/geocoding/geocode-addresses.htm) for batch geocoding of address data. This makes Masquerade an easier alternative to the our [Geocoding Toolbox](https://github.com/agrc/geocoding-toolbox).
```

**Review by steveoh — APPROVED** [review]
I like it. 

I prefer to use code blocks instead of quotes when giving instructions but that is only a stylistic choice. No need to change unless you think it's easier to read.

**Review by steveoh — APPROVED** [review]

**Comment by stdavis** [comment]
> I prefer to use code blocks instead of quotes when giving instructions but that is only a stylistic choice. No need to change unless you think it's easier to read.

I'm using quotes so that the link works. But thanks for the idea.

**Pull request merged**